pages/tech/environments.md

```diff
@@ -0,0 +1,92 @@
+import Section from '../../components/section'
+
+# Computational Environments
+
+## Overview
+
+At CarbonPlan, we use different types of computational environments to support our data science work:
+
+- **Local environments**: For day-to-day development
+- **Docker containers**: For reproducibility and deployment
+
+These environments support various activities across our workflow:
+
+| Activity        | Local Environment   | Docker                   |
+| --------------- | ------------------- | ------------------------ |
+| Development     | ✅ Primary choice   | ⚠️ Can be slower         |
+| Testing         | ✅ Quick iterations | ✅ CI integration        |
+| Deployment      | ❌ Not recommended  | ✅ Best practice         |
+| Reproducibility | ⚠️ Limited          | ✅ Excellent             |
+| Collaboration   | ⚠️ Setup required   | ✅ Consistent experience |
+
+## Local environments
+
+We primarily use two tools for managing local environments, depending on project needs.
+
+### Using Pixi
+
+[Pixi](https://pixi.sh/latest/) is our recommended tool for managing local environments, especially for projects with complex geospatial dependencies.
+
+#### Advantages of Pixi
+
+- Faster dependency resolution than conda/mamba
+- Simplified environment specification and isolation
+- Compatibility with conda-forge packages (crucial for `GDAL`, `rasterio`, etc.)
+- Deterministic builds with lockfiles
+
+#### Getting Started with Pixi
+
+Follow the [installation instructions](https://pixi.sh/latest/#installation) to set up Pixi:
+
+```bash
+# Install Pixi
+curl -fsSL https://pixi.sh/install.sh | bash
+
+# Initialize a new project
+pixi init
+
+# Add dependencies (including conda-forge packages)
+pixi add numpy pandas xarray
+pixi add -c conda-forge gdal rasterio
+
+# Run commands within the environment
+pixi run python my_script.py
+```
+
+### Alternative: Conda/Mamba
+
+For projects that benefit from the broader conda ecosystem, you can use conda or its faster alternative, mamba.
```

Comment:
@norlandrhagen, i'm curious to hear your thoughts on this section

Reply:
@andersy005 I think this is a great update! I've spent a bit of time playing with both `uv` and `pixi` this last week. In a really brief exploratory period, I think `uv` is easier to get setup and use, but still has the conda-forge source limitation. What do you think of having some sort of choice flow diagram like:


```mermaid
  graph TD;
      A[I have a conda-forge specific dep like xESMF, GDAL etc.]-->|yes| B;
      A[I have a conda-forge specific dep like xESMF, GDAL etc.]-->|no| E[Use UV];
      B[I want to stick in the conda/mamba ecosystem]-->|yes| C[Use mamba/conda-store];
      B-->|no| D[Use Pixi];

```